File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/InvalidLedgerTransactionException.java
package org.hyperledger.indy.sdk;

public class InvalidLedgerTransactionException extends IndyException
{
	private final static String message = "The ledger message is unknown or malformed.";

	InvalidLedgerTransactionException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/LedgerApi/InvalidLedgerTransactionException.cs
namespace Hyperledger.Indy.LedgerApi
{
    /// <summary>
    /// Exception thrown when attempting to send an unknown or incomplete ledger message.
    /// </summary>
    public class InvalidLedgerTransactionException : IndyException
    {
        const string message = "The ledger message is unknown or malformed.";

        /// <summary>
        /// Initializes a new InvalidLedgerTransactionException.
        /// </summary>
        internal InvalidLedgerTransactionException() : base(message, (int)ErrorCode.LedgerInvalidTransaction)
        {

        }
    }

}
